src/infra/instrumentation/instrumentation-agent.js

```diff
@@ -26,6 +26,7 @@ export default class InstrumentationAgent {
       metrics.init({
         host: process.env.HOST,
       });
+      dogapi.initialize({ api_key: process.env.DATADOG_API_KEY });
```

Comment:
You mean dogapi library? we are using two datadog libraries, because events can be written only using dogapi library.